.github/workflows/create-release.yml

```
@@ -0,0 +1,48 @@
+on:
+  push:
+    branches:
+      - master
+
+name: Upload Release Asset
+
+jobs:
+  build:
+    name: Upload Release Asset
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Set up project
+        uses:
+          actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
```

не вижу у тебя описанной матрицы, поэтому не парься и просто напиши тут 12, 14 или 15. 

```suggestion
          node-version: 14
```